Source/GSXib5KeyedUnarchiver.m

```diff
@@ -2969,6 +2985,29 @@ - (id) decodeObjectForKey: (NSString *)key
   return object;
 }
 
+- (NSInteger) decodeIntegerForKey: (NSString *)key
```

Comment:
Are you sure you need this method? When I took over this class I was trying to remove most of these specific methods and base them on decodeObjectForKey: as the super implementation will use that call and convert the result to an integer. I would even prefer to remove more of these specific type methods but for each of the remaining one there was some special issue.
I might be wrong here but this is how I remember things.

Reply:
I removed it.  It doesn't seem necessary.

Reply:
Yes, but you did not push that change. I am not able to see it.